python/ql/src/experimental/typetracking/TypeTracker.qll

```diff
@@ -0,0 +1,387 @@
+/** Step Summaries and Type Tracking */
+
+private import TypeTrackerPrivate
+
+/** Any string that may appear as the name of a piece of content. */
```

Comment:
```suggestion
/** 
 * Any string that may appear as the name of a piece of content. This will usually include things like:
 * - Attribute names (in Python)
 * - Property names (in JavaScript)
 */
```

I think that having the explanation for what "content" is here in the shared implementation, is also useful